src/SFA.DAS.EmployerAccounts.Web/Extensions/AgreementTemplateExtensions.cs

```diff
@@ -1,12 +1,12 @@
 ﻿using SFA.DAS.EmployerAccounts.Web.ViewModels;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace SFA.DAS.EmployerAccounts.Web.Extensions
 {
     public static class AgreementTemplateExtensions
```

Comment:
Are there no unit tests for this extension method?

Reply:
Nope